docs/deployments/on-demand-compute.md

```diff
@@ -0,0 +1,85 @@
+# On-demand Compute (BETA)
+
+> On-demand compute is currently in private beta. To gain early access, please sign up for the waitlist [here](https://forms.fillout.com/t/9M1VhL8Wxyus).
```

Comment:
This is the "cloud" wait list, not sure if we wan't a separate one or just ask people to get in touch by email?

Reply:
Yeah, let's just do helo@differential.dev?